integration/update_array_compat_test.go

```diff
@@ -71,9 +67,6 @@ func TestUpdateArrayCompatPop(t *testing.T) {
 			update:     bson.D{{"$pop", bson.D{{"v", int32(42)}}}},
 			resultType: emptyResult,
 		},
-		"DotNotationArray": {
```

Comment:
That case is basically the same as "DotNotation". They both check for the array path. This particularly test cannot be used in Tigris because the keyword `array` is not allowed.